src/core/stream/events_generators.ts

```diff
@@ -108,17 +108,17 @@ const EVENTS = {
              value: undefined };
   },
 
-  fullBuffer(bufferType : IBufferType) : IBufferStateFull {
-    return { type: "full-buffer",
+  fullStream(bufferType : IBufferType) : IStreamStateFull {
```

Comment:
Isn't it still full buffer that we want to know ?

Reply:
Nope. It is the RepresentationStream which has nothing left to download. The buffer itself could still need segments, such as when playing multi-period contents.